project/src/components/add-review-form/add-review-form.tsx

```diff
@@ -0,0 +1,46 @@
+import {useState} from 'react';
+import RatingItem from '../rating-item/rating-item';
+
+const RATING_ITEMS = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1];
+
+function AddReviewForm(): JSX.Element {
+  const [rating, setRating] = useState(0);
+  const [reviewText, setReviewText] = useState('');
```

Comment:
Мне кажется можно и покороче форма небольшая
```suggestion
  const [review, setReview] = useState('');
```